test/python/transpiler/test_pass_scheduler.py

```diff
@@ -589,21 +615,6 @@ def test_nested_conditional_in_loop(self):
         self.assertScheduler(self.circuit, self.passmanager, expected)
 
 
-class DoXTimesController(FlowController):
-    """A control-flow plugin for running a set of passes an X amount of times."""
-
-    def __init__(self, passes, options, do_x_times, **_):
-        super().__init__(options)
-        self.passes = passes
-        self.do_x_times = do_x_times
-
-    # pylint: disable=missing-function-docstring
-    def iter_tasks(self, metadata):
-        for _ in range(self.do_x_times(metadata.property_set)):
-            for pass_ in self.passes:
-                metadata = yield pass_
-
-
```

Comment:
This is removed because it's tested as part of `qiskit.passmanager` elsewhere, I assume?